arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/RaiseAccumulate.kt

```diff
@@ -990,6 +990,7 @@ public open class RaiseAccumulate<Error>(
     }
   }
 
+  @Suppress("NOTHING_TO_INLINE")
   public inline operator fun <A> Value<A>.getValue(thisRef: Nothing?, property: KProperty<*>): A = value
```

Comment:
This may need to be moved to allow access when RaiseAccumulate isn't available as a dispatch receiver.

Reply:
Unfortunately `getValue` is not supported with context parameters (yet).

Reply:
However, I found a way to turn it into a member of `Value<A>` itself, so we can keep the API.

Reply:
It doesn't actually need the dispatch/context, so it can just exist at the top-level